.github/workflows/ci.yml

```diff
@@ -133,6 +133,41 @@ jobs:
           command: fmt
           args: --all -- --check
 
+  benches:
```

Comment:
Thanks for adding this, it was on my TODO list for a while. Can we reduce the number of initial leaves + inserted leaves in the `azks.rs` benchmark? It looks like this stage in CI takes about 10 minutes to run, which significantly increases time-to-signal for devs. If it still takes too long we might need to tweak `parallel_vrfs.rs` as well.

Since including benchmarks in CI is primarily to detect non-compatible changes, reducing the load should be fine - devs can always tune them up locally as needed :)

Reply:
Addressed!